src/components/MovieContainer/MovieContainer.test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import { MovieContainer, mapStateToProps, mapDispatchToProps } from './MovieContainer';
+import { fetchMovies } from '../../thunks/fetchMovies.js';
+
+describe('MovieContainer', () => {
+  let wrapper
+  let mockFetch
+  let mockMovies
+
+  beforeEach(() => {
+    mockMovies = [{name: 'Aquaman'}]
+    mockFetch = jest.fn()
+    wrapper = shallow(<MovieContainer movies={ mockMovies } isLoading={ false } hasErrored={ false } fetchMovies={ mockFetch }/>)
+  }) 
+  it('should match the snapshot', () => {
+    
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  it('should render an empty div if there are no movies in props', () => {
+
+    expect(wrapper.find('div').length).toEqual(1)
+  })
+
+  describe('componentDidMount', () => {
+    it('should call fetchMovies with the correct params', () => {
+      const url = 'https://api.themoviedb.org/3/discover/movie?api_key=8d54c10134c07bfb8e01ded9e30524be&with_genres=28'
+      let mockMovies = [{name: 'Aquaman'}]
```

Comment:
Great job on this Kaylee! We'll probably have to refactor our apiKey before this gets merged in. After we pull in the other pr all we'll have to do is import the variable that is holding it into this file.